src/lib/seed.ts

```diff
@@ -41,85 +41,85 @@ export class Seed {
     host: string;
     id: string;
     git?: string | null;
-    api?: string | null;
+    addr?: string | null;
     version?: string | null;
   }) {
     assert(isDomain(seed.host), `invalid seed host: ${seed.host}`);
     assert(/^[a-z0-9]+$/.test(seed.id), `invalid seed id ${seed.id}`);
 
-    let api = null;
-    let git = null;
-    let apiPort: number | null = defaultHttpApiPort;
-    let gitPort: number | null = defaultGitPort;
+    let _seed = null;
+    let _git = null;
+    let _seedPort: number | null = defaultSeedPort;
+    let _gitPort: number | null = defaultGitPort;
```

Comment:
maybe leave a `NOTE` that `git` stuff should be merged into `seed`?

Reply:
I'll do that at the very end in #515.